flink-sql-connector-pulsar/pom.xml

```diff
@@ -44,6 +44,10 @@ under the License.
 			<artifactId>flink-connector-pulsar</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-annotations</artifactId>
```

Comment:
I hate this dependency. Can we just use `<optional>true</optional>` and document it?

Reply:
Previously we bundle this dependency from pulsar-client-all and now we add it explicit. This is not a new dependency effectively. See https://github.com/apache/pulsar/pull/19458#issuecomment-1635371635

Reply:
I see. This is the must have dependency which can't be omitted.